src/routes/prepare_game.py

```diff
@@ -1,104 +1,64 @@
 
 
-class GameDescription(BaseModel):
-    name: str
-    owner_name: str
-    difficult: int
-    players: Dict[str, str]
-    player_count: int
-    created: str
+class JoinGameOptions(BaseModel):
+    game_id: str
 
 
-@router.get(
+@router.post(
     '/create_new_game',
-    description='Create instance of the Game',
-    response_model=GameDescription
+    description='Create instance of the Game'
 )
-async def create_new_game(
-    request: Request,
-    game_config: GameConfig,
-    background_tasks: BackgroundTasks,
-):
-    game = Game()
-    current_player = request.state.player
-    background_tasks.add_task(setup_game, game)
-    gameSession = GameSession(owner_name=game_config.owner_name,
-                              owner_user_id=current_player,
-                              difficult=game_config.difficult,
-                              player_count=game_config.player_count,
-                              created=datetime.utcnow().isoformat(),
-                              game=game)
-
-    request.app.state.games.update({current_player: gameSession})
-
-    response = {'name': current_player,
-                'owner': current_player,
-                'difficult': gameSession.difficult,
-                'players': gameSession.players,
-                'player_count': f'1 of {gameSession.player_count}',
-                'created': gameSession.created}
-    return response
+async def create_new_game(request: Request, game_options: GameOptions):
+
+    # TODO bind request.state.session with controller or keep game_id in session
 
+    controller = GameController()
+    game_id = controller._ctx
+    request.app.state.games.update({game_id: controller})
+    response = {'status': 'success',
+                'game_id': game_id}
 
-def setup_game(game: Game) -> None:
-    game.setup_game()
-    logger.info('Game have setuped')
+    return response
 
 
-@router.get(
-    '/join_game/{game_name}/{character_name}',
-    description='Join to game',
-    response_model=GameDescription
+@router.post(
+    '/join_game',
+    description='Join to game'
 )
 async def join_game(
     request: Request,
-    game_name: str,
-    character_name: str
+    join_game_options: JoinGameOptions
 ):
-    current_player = request.state.player
-    gameSessions = request.app.state.games
-    gameSession = find_game(current_player, gameSessions)
-    gameSession.add_player(character_name, current_player)
-
-    response = {'name': gameSession.name,
-                'owner': gameSession.owner_user_id,
-                'difficult': gameSession.difficult,
-                'players': gameSession.players,
-                'player_count': f'{len(gameSession.players.items())} of {gameSession.player_count}',
-                'created': gameSession.created}
+    game_id = JoinGameOptions.game_id
+    try:
+        controller = find_game(request, game_id)
+        response = {'status': 'success',
+                    'game_id': controller.game_id}
```

Comment:
Game controller does not have an explicit ID attribute, but its context does:
```suggestion
        controller = find_game(request, game_id)
        response = {'status': 'success',
                    'game_id': controller._ctx['id']}
```